ios/RCTConvert+BitmovinPlayer.swift

```diff
@@ -482,7 +483,94 @@ extension RCTConvert {
             "minBitrate": adData.minBitrate
         ]
     }
-    
+
+    /**
+     Utility method to get a `BitmovinAnalyticsConfig` value from a JS object.
+     - Parameter json: JS object.
+     - Returns: The associated `BitmovinAnalyticsConfig` value or nil.
+     */
+    static func analyticsConfig(_ json: Any?) -> BitmovinAnalyticsConfig? {
+        guard
+            let json = json as? [String: Any?],
+            let key = json["key"] as? String
+        else {
+            return nil
+        }
+        var config = BitmovinAnalyticsConfig(key: key)
+        if let playerKey = json["playerKey"] as? String {
+            config = BitmovinAnalyticsConfig(key: key, playerKey: playerKey)
+        }
```

Comment:
Let's not create this config two times in case there is `playerKey`.

```suggestion
        let config: BitmovinAnalyticsConfig
        if let playerKey = json["playerKey"] as? String {
            config = BitmovinAnalyticsConfig(key: key, playerKey: playerKey)
        } else {
            config = BitmovinAnalyticsConfig(key: key)
        }
```

Reply:
881aef79c3f8080852ea79b10e0279b981df2f46